avm/res/managed-services/registration-definition/registration-assignment/main.bicep

```diff
@@ -0,0 +1,22 @@
+metadata name = 'Registration Assignment'
+metadata description = 'Create a registration assignment.'
+metadata category = 'Managed Services'
+
+@description('The ID of the registration definition.')
+param registrationDefinitionId string
+
+@description('The ID of the registration assignment.')
+param registrationAssignmentId string
+
+resource registrationAssignment 'Microsoft.ManagedServices/registrationAssignments@2019-09-01' = {
```

Comment:
As this is a different resource and not a 'child-module'  it should be placed in a file & folder `managed-services/registration-definition/modules/registrationAssignment.bicep`.

That also eliminates the need for a dedicated readme & version.json file.

Reply:
Adjusted.